app/javascript/components/components/Header.js

```diff
@@ -21,7 +24,7 @@ class Header extends Component {
                         expand="md"
                         light
                     >
-                        <NavbarBrand href="/">
+                        <NavbarBrand href={homeLink}>
```

Comment:
You may run into issues using an `href` for internal React navigation. It is recommended to use `NavLink to=` from react-router-dom